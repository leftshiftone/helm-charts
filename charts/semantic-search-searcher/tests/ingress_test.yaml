suite: test ingress
templates:
  - ingress.yaml
tests:
  - it: default
    set:
      ingress.enabled: "true"
    asserts:
      - isKind:
          of: Ingress
      - equal:
          path: metadata.name
          value: RELEASE-NAME-semantic-search-searcher-app
      - matchSnapshot:
          path: metadata.labels
      - matchSnapshot:
          path: metadata.annotations
      - matchSnapshot:
          path: spec
