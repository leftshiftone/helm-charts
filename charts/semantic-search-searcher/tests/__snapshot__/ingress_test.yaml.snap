default:
  1: |
    app.kubernetes.io/instance: RELEASE-NAME
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: semantic-search-searcher
    app.kubernetes.io/version: 0.0.0
    helm.sh/chart: semantic-search-searcher-0.0.0
  2: |
    null
  3: |
    rules:
    - host: semantic-search-searcher.local
      http:
        paths:
        - backend:
            serviceName: RELEASE-NAME-semantic-search-searcher-app
            servicePort: 80
          path: /
